src/app/components/Footer/index.jsx

```diff
@@ -59,64 +61,28 @@ const StyledParagraph = styled.p`
   margin: 0;
 `;
 
-const listItems = [
-  {
-    href: 'https://www.bbc.com/news/help-41670342',
-    id: '1',
-    text: 'Why you can trust the BBC',
-  },
-  {
-    href: 'https://www.bbc.com/terms',
-    id: '2',
-    text: 'Terms of Use',
-  },
-  {
-    href: 'https://www.bbc.com/news/help-41670342',
-    id: '3',
-    text: 'About the BBC',
-  },
-  {
-    href: 'https://www.bbc.com/news/help-41670342',
-    id: '4',
-    text: 'Privacy Policy',
-  },
-  {
-    href: 'https://www.bbc.com/terms',
-    id: '5',
-    text: 'Cookies',
-  },
-  {
-    href: 'https://www.bbc.com/news/help-41670342',
-    id: '6',
-    text: 'Accessibility Help',
-  },
-  {
-    href: 'https://www.bbc.com/news/help-41670342',
-    id: '7',
-    text: 'Contact the BBC',
-  },
-];
-
-const externalLink = {
-  href: 'https://www.bbc.co.uk/help/web/links/',
-  text: 'Read about our approach to external linking.',
-};
-
-const Footer = () => (
+const Footer = ({ list, text, link }) => (
```

Comment:
I don't think that this Footer component should include all the logic inside this React component. It should just export the styled Footer component.

This extends to all the other styled components that are inside this file - that is outside the remit of this PR however.

Furthermore, I think we should just pass [children](https://github.com/bbc/simorgh/blob/003508ff8d4e5622559e762a026e57d4a54bf40a/src/app/components/Footer/index.jsx#L66-L78) to a Footer component, rather than a Footer component holding all the logic.